api/app/serializers/v1/reading_group_serializer.rb

```diff
@@ -5,26 +5,25 @@ class ReadingGroupSerializer < ManifoldSerializer
 
     abilities
 
-    typed_attribute :name, NilClass
-    typed_attribute :privacy, NilClass
-    typed_attribute :invitation_code, NilClass
-    typed_attribute :notify_on_join, NilClass
-    typed_attribute :memberships_count, NilClass
-    typed_attribute :annotations_count, NilClass
-    typed_attribute :highlights_count, NilClass
-    typed_attribute :created_at, NilClass
-    typed_attribute :current_user_role, NilClass
-    typed_attribute :creator_id, NilClass
-    typed_attribute :texts, NilClass
-    typed_attribute :all_annotations_count, NilClass do |object, _params|
+    typed_attribute :name, Types::String
+    typed_attribute :privacy, Types::String.enum("public", "private", "anonymous")
+    typed_attribute :invitation_code, Types::String.meta(example: "E80RSYY6", unique: true)
```

Comment:
I added a new "unique" meta tag to be used from here on out for attributes that have a requirement for uniqueness  on the model.

Reply:
Sounds good.